CinebaseTests/Use Cases/Tickets/Buy/BuyTicketsPresenterSpec.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BuyTicketsPresenterSpec.swift
+//  Cinebase
+//
+//  Created by Gustavo Villar on 5/12/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Cinebase
+
+class BuyTicketsPresenterSpec: QuickSpec {
+    
+    override func spec() {
+        super.spec()
+        
+        describe("a Buy Tickets Presenter") {
+            
+            var sut: BuyTicketsPresenter!
+            var mockView: MockBuyTicketsView!
+            var mockRouter: MockRouter!
+            beforeEach {
+                mockRouter = MockRouter()
+                mockView = MockBuyTicketsView()
+                sut = BuyTicketsPresenter(view: mockView)
+                sut.dismiss = {
+                    mockRouter.isDismissed = true
+                }
+            }
+            
+            context("when a credit card is scanned") {
+                beforeEach {
+                    sut.onScanningCreditCard(
+                        withNumber: "1234",
+                        expiryMonth: 12,
+                        expiryYear: 2020,
+                        cvv: "987")
+                }
+                it("should change the description") {
```

Comment:
the test checks that the scan return is the expected one.  The description infers that you want to check if the scan returns something different than an error, what happen if the scan returns a wrong number or a error?

my other observation is, we need to check the functionality of an external library? we can check if the library is called, what do you think?